templates/jenkins/files/README.md

```diff
@@ -1,19 +1,37 @@
-# Jenkins template for Platform.sh
+# Jenkins for Platform.sh
 
-This project provides a starter kit for Jenkins projects hosted on [Platform.sh](http://platform.sh).
+This project provides a starter kit for Jenkins projects hosted on Platform.sh.
 
-We include recommended .platform.app.yaml and .platform files that should suffice for most use cases. You are free to tweak them as needed for your particular site.
+Jenkins is an open source automation server written in Java. Jenkins helps to automate the non-human part of the software development process, with continuous integration and facilitating technical aspects of continuous delivery.
 
-## Platform.sh configuration files
+## Services
+
+* Java 8
+
+## Post-install
+
+1. After running a Jenkins instance by Platform.sh, the post-installation setup wizard begins.
```

Comment:
"running a Jenkins instance by Platform.sh" - The grammar here is a bit clunky.  I'm not sure what is intended here.  Maybe "After creating a Jenkins project"?  In the others I've specifically called out the first-run behavior.  Is that the case here?

Reply:
After creating a Jenkins project is fine.